public/docs/ts/latest/guide/attribute-directives.jade

```diff
@@ -25,12 +25,12 @@ a#directive-overview
 
   There are three kinds of directives in Angular:
   
-  1. Components &mdash; directives with a template.
-  1. Structural directives &mdash; change the DOM layout by adding and removing DOM elements.
-  1. Attribute directives &mdash; change the appearance or behavior of an element, component, or another directive.
+  1. Components&mdash;directives with a template.
+  1. Structural directives&mdash;change the DOM layout by adding and removing DOM elements.
+  1. Attribute directives&mdash;change the appearance or behavior of an element, component, or another directive.
 
   *Components* are the most common of the three directives.
-  You saw a component for the first time in the [QuickStart](../quickstart.html) example.
+  You saw a component for the first time in the [QuickStart](../quickstart.html) guide.
```

Comment:
I thought that we were to use the term "page" instead of "guide" (although, in this case, the original "example" seems to be ok too).